app/src/main/res/values-ta/strings.xml

```diff
@@ -1,2 +1,305 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources></resources>
\ No newline at end of file
+<resources xmlns:tools="http://schemas.android.com/tools" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="app_name">கேடிமா</string>
+    <string name="action_search">தேடல்</string>
+    <string name="action_add">கூட்டு</string>
+    <plurals name="selectedCardCount">
+        <item quantity="one">&lt;xliff: g&gt;%d &lt;/xliff: g&gt; தேர்ந்தெடுக்கப்பட்டது</item>
```

Comment:
@TamilNeram You seem to have turned some `<` into `&lt;`, this causes invalid translations. I had to remove them to prevent the app from crashing. Please be aware of this while translating :)